meetings/2021-11-17.md

```diff
@@ -0,0 +1,189 @@
+In order to avoid complex problems of privacy of access rules, Solid led people to limit the ability to see the rules. 
+But that also limits the systems that can be deployed. 
+See [minimal credentials disclosure use case](https://solid.github.io/authorization-panel/authorization-ucr/#uc-minimalcredentials) to see the problem that we want to be able to deal with. 
+Where either resources are private to owner of the box or need to be public. 
+I want it to be possible where people have different WebID, different Credentials etc.
+* **Fred**: Even today I can have 100 email addresses, I can use them to authenticate to different systems. 
+I never assume that I will use my personal ID when I need my work.
+* ...: For authentication purposes I will just have one WebID to make my life simple. 
+If I have 5 i need to decide which one I use where.
+* **Justin**: I think it would be useful to have your (Henry's) approach written down. 
+This is good discussion which should be captured.
+* **Fred**: You could make group of WebIDs and use that to grant access. With that you wold just use group to that access.
+* **Henry**: yes, that works on systems we use today. 
+When you go to Twitter you know what ids you have used there, 
+but on open systems like we want to build people could create rules that allow people access under Identities or credentials that the user's App does not know about in advance.
+* **Matthieu**: I find an idea of virtual hierarchies interesting. 
+Do you have them documented
+* **Fred**: In issue 102 I have it documented.
+* **Matthieu**: I refer how do you map containment to uri paths.
+* **Fred**: Each user has their own named graph. 
+For each resource which needs URI path we include tag predicate. 
+Our system looks at path uri and looks for the resource using that tag.
+* **Matthieu**: It would be interesting to fill it back in the solid protocol. 
+Solid is very undescriptive how to deal with containment.
+* **Fred**: Are you suggesting that we propose it to solid spec?
+* **Matthieu**: I think this would be great. 
+I see this problem very important.
+* **Fred**: Where I should explain it?
+* **Matthieu**: solid-specification repo
+* **Justin**: +1
+* **Pavlik**: If resource is in many containers, how do you do inheritance of access rules?
+* **Fred**: We deal with that by doing union to define ACL if one of them gives them than they get it.
+* **Matthieu**: So ALC is still linked to the resource not it virtual hierarchy?
+* **Fred**: ACL can be on the resource or on the container.
+* **Matthieu**: There is ACL for one URI
+* **Fred**: We have ACL object and we don't limit how many ACLs you can apply to a single resource, we just put them together. They are applied to the permanent URI. 
+Paths are just public facing phenomena. 
+Those are dynamically generated on the fly. 
+Even if you move container the permissions would stay with it since they are connected to permanent uri.
```

Comment:
It sounds like permanent URI is serving like inode... Might be worth stating such.
```suggestion
Even if you move container, the permissions would stay with it, since they are connected to permanent URI.
```